pynicotine/gtkgui/widgets/treeview.py

```diff
@@ -416,11 +468,29 @@ def get_user_status_tooltip_text(column_value, _strip_prefix):
     def show_user_status_tooltip(self, pos_x, pos_y, tooltip, column):
         return self.show_tooltip(pos_x, pos_y, tooltip, column, ("status",), self.get_user_status_tooltip_text)
 
+    @staticmethod
+    def get_country_tooltip_text(column_value, strip_prefix):
+
+        if not column_value.startswith(strip_prefix):
+            return _("Unknown")
```

Comment:
![Screenshot_2022-11-06_20-36-49](https://user-images.githubusercontent.com/88614182/200194042-2957418f-c241-4c53-bb22-ec843ca4b816.png)

Reply:
The code was previously writing an invalid country value to the config (only the country code instead of flag_ + country code). Should be fixed when the country updates again.

Reply:
Yes, it does fix when the user comes on line. The problem did not appear after restart even as the user was offline at startup.